src/styles/components/cards.scss

```diff
@@ -0,0 +1,119 @@
+.stars-line {
+  display: flex;
+}
+
+.stars-line__star {
+  width: 16px;
+  height: 16px;
+  background-image: url(../images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+
+  &:nth-child(-n + 4) {
+    background-image: url(../images/star-active.svg);
+  }
+}
+
+.card {
+  box-sizing: border-box;
```

Comment:
It's the general rule that should be applied for all elements, let's set it using `*` selector